website/docs/r/storage_bucket_iam.html.markdown

```diff
@@ -38,6 +40,22 @@ resource "google_storage_bucket_iam_member" "member" {
 }
 ```
 
+## google\_storage\_bucket\_iam\_policy
+
+When applying a policy that does not include the roles listed below, you lose the default permissions which google adds to your bucket:
+* `roles/storage.legacyBucketOwner`
+* `roles/storage.legacyBucketReader`
+
+If this happens only an entity with `roles/storage.admin` privileges can repair this bucket's policies. It is recommended to include the above roles in policies to get the same behaviour as with the other two options.
+
+```hcl
+resource "google_storage_bucket_iam_policy" "member" {
+  bucket = "your-bucket-name"
+  policy_data = "policy_data"
```

Comment:
Looks like a string snuck in here- I think you meant to have this refer to a policy data source.

Reply:
Done.